lib/graphql/subscriptions.rb

```diff
@@ -118,14 +116,23 @@ def execute(subscription_id, event, object)
       nil
     end
 
+    # Run the update query for this subscription and deliver it
+    # @see {#execute_update}
+    # @see {#deliver}
+    # @return [void]
+    def execute(subscription_id, event, object)
+      res = execute_update(subscription_id, event, object)
+      deliver(subscription_id, res)
+    end
```

Comment:
This actually _unbreaks_ the API for 1.11: old code will still work; `execute_update` _can be_ called by implementations that want to separate execution and delivery.